pkg/resources/user/userHelper.go

```diff
@@ -125,14 +125,23 @@ func GetUsersInActiveIDPs(ctx context.Context, serverClient k8sclient.Client) (*
 		}
 	}
 
+	clusterIdentities := &usersv1.IdentityList{}
+	err = serverClient.List(ctx, clusterIdentities)
+	if err != nil {
+		return nil, errors.Wrapf(err, "could not list cluster identities")
+	}
+
 	activeUsers := &usersv1.UserList{}
 
 	//go over each user
 	for _, user := range openshiftUsers.Items {
 		// get  their identities - can be multiple?
-		identities, err := GetIdentities(ctx, serverClient, user)
+		identities, err := GetIdentities(user, clusterIdentities)
 		if err != nil {
-			return nil, errors.Wrapf(err, "could not get identities for user %v", user.Name)
+			// it is possible for user CRs to exist without any identity CR's
+			logger.Warning(fmt.Sprintf("could not get identities for user %v", user.Name))
+			activeUsers.Items = append(activeUsers.Items, user)
+			continue
```

Comment:
Maybe I am missing something, but essentially, is there any difference now between the `openshiftUsers` and the `activeUsers` ?

Reply:
The `openshiftUsers` are the list of user that have accounts in sso and are cluster users. While the  `activeUsers` have accounts in sso, are cluster users and have an active IDP. They are a bit more refined.

Reply:
After the refactor yes, initially, we were returning all users, regardless of whether their identity is found or not. But it looks like the logic got updated. Thanks